goth/configuration.py

```diff
@@ -292,6 +298,10 @@ def load_yaml(
             name = node["name"]
             type_name = node["type"]
             use_proxy = node.get("use-proxy", False)
+
+            payment_config_name = node.get("payments", DEFAULT_PAYMENT_CONFIG_NAME)
```

Comment:
I find the name `payments` ambiguous in this context. How about `payment_platform` or `payment_config`?